doc/mk_sdcard_image.sh

```diff
@@ -13,8 +13,8 @@ usage()
 	echo "`basename "$0"` <-p image-folder> <-d image-file> [-s image-size-gb] [-u dom0|domd|doma]"
 	echo "	-p image-folder	Base daily build folder where artifacts live"
 	echo "	-d image-file	Output image file or physical device"
-	echo "	-s image-size	Optional, image size in GB"
-	echo "  -u domain	Optional, unpack the domain specified"
+	echo "	-s image-size	Optional, image size in GiB"
```

Comment:
how is this change relevant to the rest of the changes in this commit?
Please send a dedicated clean up patch